src/util/MongoInstance.ts

```diff
@@ -54,7 +54,14 @@ export default class MongodbInstance {
       this.opts.binary.debug = this.opts.debug;
     }
 
-    if (this.opts.instance && this.opts.instance.debug) {
+
+    this.debug = (msg: string) : void => {
+      if (this.opts.instance && this.opts.instance.debug) {
+        console.log(`Mongo[${this.opts.instance && this.opts.instance.port}]: ${msg}`);
+      }
+    };
+
+    /*if (this.opts.instance && this.opts.instance.debug) {
```

Comment:
@nodkz By commenting out this part, I provoked warning message in the test suite : 
```bash
●  Cannot log after tests are done. Did you forget to wait for something async in your test?
    Attempted to log "Mongo[65156]: CLOSE: 0".
```
Could you explain me a bit more what are we doing in this condition ?

Reply:
It looks like somewhere unproperly was killed instance – `kill()` method.
The problem may be in it. Or maybe in Killer, which kill instance on shutdown the main script. Need to debug closely. Better to uncomment this thing to fix it in the future and not forget about it.

Reply:
@nodkz Actually I think we just forgot to add an await to the MongoMemoryServer.stop() calls in the afterAll of our singleDB and multipleDB test  : 

```javascript
afterAll(() => {
  if (con1) con1.close();
  if (con2) con2.close();
  if (mongoServer1) mongoServer1.stop();
  if (mongoServer2) mongoServer2.stop();
});
```
Adding an async/await solved the warnings in the tests (without removing the commenting on the condition above) : 
```javascript
afterAll(async () => {
  if (con1) con1.close();
  if (con2) con2.close();
  if (mongoServer1) await mongoServer1.stop();
  if (mongoServer2) await mongoServer2.stop();
});
```

Reply:
👍